Formula/kubectl-kruise.rb

```diff
@@ -0,0 +1,37 @@
+class KubectlKruise < Formula
```

Comment:
Maybe we should add the formula in https://github.com/Homebrew/homebrew-core/tree/main/Formula/k, so that we can use `brew install kubectl-kruise` to install it.